python/nav/models/sql/changes/sc.05.12.0001.sql

```diff
@@ -0,0 +1,20 @@
+-- This migration is to ensure that for accounts that don't have a default
+-- dashboard we set a default dashboard
```

Comment:
Just leaving a quick comment before the weekend: This is the weakness of NAV's homegrown migration system:  This migration file will conflict with #3231.  One of us needs to change our PR accordingly :)

Reply:
Indeed, let's see who it will be :sweat_smile: